game/src/test/scala/io/aigar/HelperSpec.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar
+
+import com.mchange.v2.c3p0.ComboPooledDataSource
+import slick.driver.H2Driver.api._
+
+trait HelperSpec {
+  def getDatabase() = {
```

Comment:
So this returns an in-memory DB for testing purposes, correct? If this is the case, could we rename the function to indicate that?

E.g. `getInMemoryDatabase` or `getTestDatabase`